srcpkgs/runit-void/template

```diff
@@ -32,10 +32,12 @@ post_install() {
 	vmkdir usr/bin
 	mv ${DESTDIR}/usr/sbin/* ${DESTDIR}/usr/bin
 	vconf ${FILESDIR}/hostname
-	vconf ${FILESDIR}/os-release
 	vconf ${FILESDIR}/locale.conf
 	vinstall ${FILESDIR}/apparmor 644 /etc/default/
 	vinstall ${FILESDIR}/09-apparmor.sh 644 /etc/runit/core-services/
+	vmkdir usr/lib
+	vinstall ${FILESDIR}/os-release 644 /usr/lib/
+	ln -s ../usr/lib/os-release ${DESTDIR}/etc/os-release
```

Comment:
Is this the norm?

Reply:
It's not necessarily how I would have coded it, but I was trying to stay as close to the existing structure as possible.

Reply:
If you're referring to the "symlink to /usr/lib/" thing, then yes, that's explicitly part of the [spec](https://www.freedesktop.org/software/systemd/man/os-release.html).

Reply:
I mean putting it under `usr/lib` instead of directly in `etc`

Reply:
The relevant section:

> `/usr/lib/os-release` is the recommended place to store OS release information as part of vendor trees. `/etc/os-release` should be a relative symlink to `/usr/lib/os-release`, to provide compatibility with applications only looking at `/etc/`.